src/time/Storage.cpp

```diff
@@ -4,29 +4,46 @@
 
 namespace precice::time {
 
+const double Storage::WINDOW_START = 0.0;
+
+const double Storage::WINDOW_END = 1.0;
+
 Storage::Storage()
     : _sampleStorage{}
 {
 }
 
 void Storage::initialize(Eigen::VectorXd values)
 {
-  _sampleStorage.emplace_back(std::make_pair(0.0, values));
-  _sampleStorage.emplace_back(std::make_pair(1.0, values));
+  _sampleStorage.emplace_back(std::make_pair(WINDOW_START, values));
+  _sampleStorage.emplace_back(std::make_pair(WINDOW_END, values));
```

Comment:
These kind of changes we could already directly merge into develop to reduce the diff. Ok @uekerman?

Reply:
sure

Reply:
Done. 1e6a41ec6d7178b84f8f4ac6e726af26507ee2ae